runtime/vm/createramclass.cpp

```diff
@@ -2169,7 +2169,7 @@ internalCreateRAMClassDone(J9VMThread *vmThread, J9ClassLoader *classLoader, J9C
 		}
 
 #if defined(J9VM_OPT_JFR)
-		hostClassLoader->loadedClassCount += 1;
+		javaVM->loadedClassCount += 1;
```

Comment:
This should be done atomically.

Reply:
The calling thread should be holding the classtable mutex so we don't need atomic?

Reply:
It seems you're right, however, it would help if this function called out that expectation (like line 759 does for `computeVTable()`.